### DISCUSSION TITLE
Bug reporting question

### DISCUSSION THREAD

**Discussion by RobCranfill** [wmerkens/vscode-circuitpython#10] [General]
(Thanks for taking this on! I love using VSC for developing in CP on various MCUs, almost all from Adafruit, and it's great to see this being supported better!)

I would love to contribute as I can, which for now is perhaps just bug/feature reports, but I am a little unsure as to just how to document any issues I find. Is there a log file I should include?

For example, I found a problem with the "Update All Libraries" command, but I don't seem to be able to copy the error text from the dialog.... is a screenshot useful?

![image](https://github.com/user-attachments/assets/b0148aa0-88f2-4030-8693-d198ada0c1d7)

I am just posting that here as an example; if appropriate I will open a bug report with that, or look forward to any suggestions.

 /rob

**Comment by wmerkens** [comment]
This is fine; you can file a report using the Issue option. I also came across this a few days ago and will look into it. I fell back to using circup at the CLI.

**Comment by RobCranfill** [comment]
I did find perhaps the right log at 

/home/rob/.config/Code/logs/20241209T101407/window2/exthost/exthost.log

I'm comfortable with using Circup, too, but just like checking out new (to me) features! :-)

**Comment by wmerkens** [comment]
Thanks
